Problem3.rb

```diff
@@ -0,0 +1,32 @@
+#check for duplicates from from positive integers array
+include Math
+
+
+def ContainsDuplicates numbers
+
+  if numbers.nil?
+     raise ArgumentError, 'Argument is null' 
+  end 
+
+  numbers.each   do |number|
+
+    if !number.is_a? Integer
+      raise ArgumentError, 'Argument is not numeric' 
+    end
+
+    if !number.positive?

```

Comment:
This is the perfect example why this checks should not exist :)

Please try to run your code with the following input [2, 1, 1].